libs/cms/src/lib/loaders/organizationLogo.loader.ts

```diff
@@ -0,0 +1,28 @@
+import DataLoader from 'dataloader'
+import { Injectable } from '@nestjs/common'
+
+import { NestDataLoader } from '@island.is/nest/dataloader'
+import { CmsContentfulService } from '../cms.contentful.service'
+
+export type LogoUrl = string | null
+
+export type OrganizationLogoDataLoader = DataLoader<string, LogoUrl, string>
+
+@Injectable()
+export class OrganizationLogoLoader implements NestDataLoader<string, LogoUrl> {
+  constructor(private readonly cmsContentfulService: CmsContentfulService) {}
+
+  async loadOrganizationLogo(
+    organizationTitles: readonly string[],
+  ): Promise<Array<LogoUrl>> {
+    const organizationLogos = await this.cmsContentfulService.getOrganizationLogos(
+      organizationTitles.map((title) => title),
```

Comment:
Redundant map.

Reply:
its because its readonly which is required when instantiating new DataLoader. 
<img width="1088" alt="image" src="https://user-images.githubusercontent.com/112904566/199066896-d829888f-5301-4ec0-ba73-5347943241ed.png">

Do you have a better suggestion? @eirikurn

Reply:
I just updated it to use type assertion instead.
```
    const organizationLogos = await this.cmsContentfulService.getOrganizationLogos(
      organizationTitles as string[],
    )
```